tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -251,6 +251,18 @@ Install Anaconda:
 Follow the instructions on the [Anaconda download
 site](https://www.continuum.io/downloads).
 
+Note: If tensorflow has been installed via pip previously, then one should 
```

Comment:
Do you mean "has been installed via pip outside the Anaconda environment previously"? If so, it will be good to make it explicit.